Source/gs/GSHandler.cpp

```diff
@@ -995,6 +995,60 @@ bool CGSHandler::GetCrtIsFrameMode() const
 	return smode2.ffmd;
 }
 
+std::pair<uint64, uint64> CGSHandler::GetCurrentDisplayInfo()
+{
+	std::lock_guard<std::recursive_mutex> registerMutexLock(m_registerMutex);
+	unsigned int readCircuit = GetCurrentReadCircuit();
+	DISPFB fb;
+	DISPLAY d;
+	switch(readCircuit)
+	{
+	case 0:
+		fb <<= m_nDISPFB1.value.q;
```

Comment:
No need to set the fb and d variables here, we can just return the registers values directly.
Also, we can add default above case 0 to make sure we always return something and avoid a compiler warning.

Reply:
haha, i actually spent way too long trying to understand why this left shift assignment doesnt always return zero.... until i realized `convertible` is not part of the standard.

Reply:
Yeah, sorry, kinda abusing C++ operators overloading here 😄

Reply:
why not just overload assignment operator?

Reply:
Don’t remember why. Maybe I wanted to prevent unwanted assignments?